src/components/Road.vue

```diff
@@ -25,12 +26,19 @@
           <v-icon>close</v-icon>
         </v-btn>
         <v-card-title>
-          <h1>I am a...</h1>
+          <h1>Road Settings</h1>
         </v-card-title>
         <v-card-text>
           <v-select
             v-model="year"
             :items="[{value: 0,text:'First Year/Freshman'},{value: 1,text:'Sophomore'},{value:2,text:'Junior'},{value:3,text:'Senior'},{value:4,text:'Super Senior'}]"
+            label="I am a..."
+            outlined
+          />
+          <v-switch
```

Comment:
This only seems to be hiding IAP for me after a page refresh (presumably it's setting the cookie correctly, but something else is going wrong in the meantime).  Unhiding IAP does seem to work though.  This may be because it is not a boolean?